src/atoms/Heading/Heading.web.js

```diff
@@ -1,5 +1,111 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { getColorKeys, getColor } from '../../_helpers/theme';
+import baseTheme from '../../tokens/theme';
+import automation from '../../_helpers/automation-attributes.web';
```

Comment:
will change `automation-attributes.web` to `automation-attributes` this after the issue is resolved.